openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1079,74 +1101,88 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
 
-If the Client requested an encrypted response, the Batch Credential Response MUST be sent as a JWT using the parameters from the `credential_response_encryption` object and using the `application/jwt` media type. If encryption was requested in the Batch Credential Request and the Batch Credential Response is not encrypted, the Client SHOULD reject the Credential Response.
+If all requests are responded to with a `transaction_id`, the Issuer MUST use the HTTP status code 202 (as detailed in Section 15.3.3 of [@!RFC9110]). If not, the HTTP status code 200 MUST be used for a successful Batch Credential Response.
 
+If the Client requested an encrypted response, the Batch Credential Response MUST be sent as a JWT using the parameters from the `credential_response_encryption` object and using the `application/jwt` media type. If encryption was requested in the Batch Credential Request and the Batch Credential Response is not encrypted, the Client SHOULD reject the Credential Response.
 If the Batch Credential Response is not encrypted, it MUST be sent as a JSON object using the `application/json` media type.
 
-The following parameters are used in the Batch Credential Response:
+The following parameters are used in the JSON-encoded Batch Credential Response body:
 
-* `credential_responses`: REQUIRED. Array that contains Credential Response objects, as defined in (#credential-response), and/or Deferred Credential Response objects, as defined in (#deferred-credential-response), with the exception of `c_nonce` and `c_nonce_expires_in`. Each element within the array matches the corresponding Credential Request object by array index in the `credential_requests` parameter of the Batch Credential Request.
-* `c_nonce`: OPTIONAL. The `c_nonce` as defined in (#credential-response). 
+* `credential_responses`: REQUIRED. Array that contains Batch Credential Response objects each corresponding to every Batch Credential Request objects. Each element within the array matches the corresponding Batch Credential Request object by array index in the `credential_requests` parameter. A Batch Credential Response object contains either a `credentials` object (and possibly a `notification_id`) or a `transaction_id` parameter, but not `c_nonce` and `c_nonce_expires_in` parameters. Each element may refer to a different Credential Configuration or the same Credential Configuration with a different Credential Dataset.
+  * `credentials`: REQUIRED if `transaction_id` is not present within a particular Batch Credential Response object. The `credentials` as defined in (#credential-response).
+  * `notification_id`: OPTIONAL. The `notification_id` as defined in (#credential-response).
+  * `transaction_id`: REQUIRED if `credentials` is not present within a particular Batch Credential Response object. The `transaction_id` as defined in (#credential-response).
+* `c_nonce`: OPTIONAL. The `c_nonce` as defined in (#credential-response).
```

Comment:
As above, the response data structure should be re-used from the single-credential endpoint.

Reply:
The Batch Response objects do not contain `credential`, `c_nonce` and `c_nonce_expires_in`